flow/data_pipeline/query.py

```diff
@@ -114,13 +114,12 @@
         speed,
         acceleration,
         road_grade,
-        GREATEST(0, 2041 * speed * ((
-            CASE
-                WHEN acceleration > 0 THEN 1
-                WHEN acceleration < 0 THEN 0
-                ELSE 0.5
-            END * (1 - {0}) + {0}) * acceleration + 9.807 * SIN(road_grade)
-            ) + 2041 * 9.807 * 0.0027 * speed + 0.5 * 1.225 * 3.2 * 0.4 * POW(speed,3)) AS power,
+        GREATEST(0, 2041 * acceleration * speed +
+            3405.5481762 +
+            83.12392997 * speed +
```

Comment:
this probably doesn't matter, but the C_1 in the document is 83.123929917, there is missing one towards the end